specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.0/keyvault.json

```diff
@@ -3887,6 +3887,11 @@
           },
           "readOnly": true,
           "x-nullable": false
+        },
+        "hsmPlatform": {
```

Comment:
FYI, The current design for different hsm platform scenario
---
![image](https://user-images.githubusercontent.com/21369396/235230695-787f8d43-4e20-487f-b07b-ff31b2d0dee3.png)